schema/schema_merge.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/hashicorp/hcl-lang/lang"
 	"github.com/hashicorp/hcl-lang/schema"
 	"github.com/hashicorp/hcl/v2"
-	"github.com/hashicorp/terraform-json"
+	tfjson "github.com/hashicorp/terraform-json"
```

Comment:
This should work just fine without the explicit alias, I think?

Reply:
it was my editor so must be a goimports/gofmt thing